### ISSUE TITLE
Recurring transaction triggers each time I log in, even if that day's transaction exists already

### ISSUE THREAD

**Issue by taylerhow** [Ledger-Software/Ledger#282]
I logged in 4 times and got 4 duplicates of the same recurring transaction. It should only be recorded once daily.

![image](https://cloud.githubusercontent.com/assets/10555067/25747290/0044b5c6-3175-11e7-86e5-360731e2c702.png)

**Comment by shellajt** [comment]
I cannot recreate this. Can somebody else confirm for me?

**Comment by taylerhow** [comment]
@shellajt Pulled down master and made a new DB, seems to be okay. However, the one transaction that was added did not have the right date. I made a daily recurrence from May 3 - May 10, logged out, & logged back in. One transaction was added with the date May 4, but it is currently May 6 5:44 AM EST. Logged out multiple times afterwards and no duplicates were created, so that's good.

Could've been on that one DB that got super borked during testing, but before I noticed. Sorry :(
